appAndroid/feed/src/main/java/com/gchristov/newsfeed/feed/FeedActivity.kt

```diff
@@ -142,6 +171,39 @@ private fun FeedState(
     }
 }
 
+@Composable
+private fun MainAppBar(
+    searchWidgetState: SearchWidgetState,
+    searchTextState: String,
+    onTextChange: (String) -> Unit,
+    onCloseClicked: () -> Unit,
```

Comment:
Nitpick: We could prefix this with `onSearch` so that it's clear what they refer to.

Reply:
Next PR will tackle this too